src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
+import org.hibernate.SessionFactory;
 
 public class ActorServiceImpl implements ActorService {
+    private SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+    private ActorDao actorDao = new ActorDaoImpl(sessionFactory);
+
+    public ActorServiceImpl(ActorDao actorDao) {
```

Comment:
This constructor is not used because the `ActorDao` instance is created directly in this class. After you remove the `actorDao` initialization from line 13, this constructor will be used to inject the `ActorDao` dependency.